lib/models/project.js

```diff
@@ -300,20 +320,6 @@ class Project {
   }
 
   /**
-    Resolves the absolute path to a file.
-
-    @private
-    @method resolve
-    @param  {String} file File to resolve
-    @return {String}      Absolute path to file
-   */
-  resolve(file) {
```

Comment:
Thoughts on adding a private API deprecation here also?  Might not matter though because I didn't find any usages of `project.resolve` in emberobserver...

Reply:
I looked too, I couldn't find anything. I vote for leaving it out and you can assign the future issue to me when something blows up. :-)

Reply:
Sounds good to me!